services/QuillLMS/spec/models/segment_integration/user_spec.rb

```diff
@@ -3,45 +3,51 @@
 require 'rails_helper'
 
 RSpec.describe SegmentIntegration::User do
-  let(:teacher) { create(:teacher) }
+  let(:teacher1) { create(:teacher) }
+  let(:teacher2) { create(:teacher, flags: ["private", "beta"]) }
```

Comment:
Super-minor, but do you need to create a second teacher instead of just setting `flags` on the original model?  It doesn't look like there are any `it` cases below where you want to check both models and see different behavior, so you could probably just make this one model.  It doesn't really matter, though, so no need to clean it up.

Reply:
I thought about suggesting this too.  On second thought, I think it should be cleaned up.

Having teacher1 and teacher2 is frequently an indicator that you'll have a test that uses both teachers in the same spec.